src/upload-tests.proj

```diff
@@ -102,8 +102,19 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="CopyTestILCToTestHostDir" Condition="'$(BuildingUAPAOTVertical)' == 'true'">
+    <PropertyGroup>
+      <_TestILCFolderCopy>$(TestSharedFXDir)TestILC/</_TestILCFolderCopy>
```

Comment:
Why are we using TestSharedFxDir for this? Perhaps we need to create a "ArchivePayloadRootDir" or something and set it up accordingly.